src/js/views/patients/worklist/flow_views.js

```diff
@@ -98,7 +101,7 @@ const FlowItemView = View.extend({
     this.showChildView('check', checkComponent);
   },
   showState() {
-    if (!this.model.isDone()) {
+    if (!this.model.isDone() || !this.canEdit) {
```

Comment:
the flow already has a read only version of state.. so we can follow this pattern or this could be 
```js
if (!this.model.isDone() || !this.canEdit) return;
```
where the state is rendered inside the region in the template.